integrations/openai/integration.definition.ts

```diff
@@ -1,15 +1,30 @@
 import { IntegrationDefinition, interfaces, z } from '@botpress/sdk'
-import { modelId } from 'src/schemas'
+import { languageModelId } from 'src/schemas'
 
 export default new IntegrationDefinition({
   name: 'openai',
-  version: '2.2.2',
+  version: '2.3.0',
   readme: 'hub.md',
   icon: 'icon.svg',
   entities: {
     modelRef: {
       schema: z.object({
-        id: modelId,
+        id: languageModelId,
+      }),
+    },
+    imageModelRef: {
+      schema: z.object({
+        id: z.string(),
+      }),
+    },
+    imageGenerationParams: {
+      schema: z.object({
+        id: z.string(), // TODO: remove
+        style: z
+          .enum(['natural', 'vivid'])
+          .default('vivid')
```

Comment:
**question(non-blocking):** is this a real default or a placeholder ?

Reply:
A default, same default value that OpenAI uses if this param is not provided.